KinConnections/db_wrapper.py

```diff
@@ -113,6 +113,7 @@ def get_all_connectors():
     all_approved = []
     for connector in all:
         if(connector.get('approved', None)):
+            connector["education"] = ""
```

Comment:
Had to make this change for the time being due to the education field having a different format for different connectors. Here's an example of what I was seeing:
```
MBA @ Harvard | Engineering @ UT
{"School0Name" : "UT Austin", "School0Major" : "ECE"}
```
Let's figure out which format we want to stick with before merging this so I can make the appropriate changes to the filtering option.

Reply:
Going with a simple string format.